src/preferences/dialog/dlgprefeffects.cpp

```diff
@@ -205,19 +205,20 @@ void DlgPrefEffects::effectsTableItemSelected(const QModelIndex& selected) {
     effectType->setText(EffectsBackend::translatedBackendName(pManifest->backendType()));
 }
 
-void DlgPrefEffects::slotChainPresetSelected(const QModelIndex& selected) {
+void DlgPrefEffects::slotChainPresetSelectionChanged(const QItemSelection& selected) {
     VERIFY_OR_DEBUG_ASSERT(m_pFocusedChainList) {
         return;
     }
     // Clear the info box and return if the index is invalid, e.g. after clearCurrentIndex()
     // in eventFilter()
-    if (!selected.isValid() ||
-            m_pFocusedChainList->selectionModel()->selectedRows(0).count() > 1) {
+    auto* pSelModel = m_pFocusedChainList->selectionModel();
+    auto selIndices = pSelModel->selectedIndexes();
+    if (!pSelModel->hasSelection() || selIndices.count() > 1) {
```

Comment:
why `||`?

Reply:
This should clear the info labels `if (selIndices.count() != 1)`.

I pushed the simplified version. Thanks!